### PR TITLE
'jf setup' command UX improvements

### PR DESCRIPTION
**Pull request by eyalbe4**
- [x] All [tests](https://github.com/jfrog/jfrog-cli-core#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli-core/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----
* Fix a bug, causing the "progress message" to "visually refresh"
* Improve messages text
* Add a message, requesting the user to verify his email
* Change the time before the browser opens from 5 to 8 seconds

### CODE REVIEW

File: general/envsetup/envsetup.go [hunk 1]
```diff
@@ -128,7 +132,11 @@ func (ftc *EnvSetupCommand) getNewServerDetails() (serverDetails *config.ServerD
 
 		// Wait for 'ready=true' response from MyJFrog
 		if resp.StatusCode == http.StatusOK {
-			ftc.progress.SetHeadlineMsg("Ready for your DevOps journey? Please hang on while we create your environment")
+			if !readyMessageDisplayed {
+				ftc.progress.ClearHeadlineMsg()
+				ftc.progress.SetHeadlineMsg("Almost done! Please hang on while JFrog CLI completes the setup")
```
Comment (talarian1) [inline general/envsetup/envsetup.go:137]: This message is a bit inaccurate, we are waiting here for a "ready" response from the website side...

File: general/envsetup/envsetup.go [hunk 2]
```diff
@@ -50,8 +50,8 @@ func NewEnvSetupCommand(url string) *EnvSetupCommand {
 }
 
 func (ftc *EnvSetupCommand) Run() (err error) {
-	ftc.progress.SetHeadlineMsg("To complete your JFrog environment setup, please fill out the details in your browser")
-	time.Sleep(5 * time.Second)
+	ftc.progress.SetHeadlineMsg("Just fill out its details in your browser")
```
Comment (talarian1) [inline general/envsetup/envsetup.go:53]: The new message sounds a bit out of context to me.

**Review by talarian1 — APPROVED** [review]

File: general/envsetup/envsetup.go (hunk 1 above)
Comment (eyalbe4) [reply general/envsetup/envsetup.go:137]: I agree that the message not super specific, but I wouldn't say it's incorrect. The user shouldn't be concerned with the MyJFrog integration details.

File: general/envsetup/envsetup.go (hunk 2 above)
Comment (eyalbe4) [reply general/envsetup/envsetup.go:53]: The message would indeed be out of context withput this mew PR in jfrog-cli - https://github.com/jfrog/jfrog-cli/pull/1487
The above PR adds the following lines before the above message - 
```
	fmt.Println(coreutils.PrintTitle("Thank you for installing JFrog CLI! 🐸"))
	fmt.Println(coreutils.PrintTitle("We'll now set up a FREE JFrog environment in the cloud for you, and configure your local machine to use it."))
	fmt.Println("Your environment will be ready in less than a minute.")
```
So this is how it looks like together - 
image

**Pull request merged**